mentoring/mentoring.py

```diff
@@ -36,42 +34,26 @@
 from xblock.fields import Boolean, Scope, String, Integer, Float, List
 from xblock.fragment import Fragment
 
-from .light_children import XBlockWithLightChildren
-from .title import TitleBlock
-from .header import SharedHeaderBlock
-from .message import MentoringMessageBlock
-from .step import StepParentMixin
-from .utils import loader
+from components.title import TitleBlock
+from components.header import SharedHeaderBlock
+from components.message import MentoringMessageBlock
+from components.step import StepParentMixin, StepMixin
```

Comment:
@bradenmacdonald I believe these all were imported into `components` namespace. Even if it's not it might make sense to do so, so that this block would become more semantical:

    from components import TitleBlock, SharedHeaderBlock, MentoringMessageBlock, ...